test/Interpreter/objc_class_properties_runtime.swift

```diff
@@ -2,16 +2,14 @@
 
 // RUN: %clang -arch %target-cpu -target %target-cpu-apple-macosx10.11 -isysroot %sdk -fobjc-arc %S/Inputs/ObjCClasses/ObjCClasses.m -c -o %t/ObjCClasses.o
 
-// RUN: %swiftc_driver -target $(echo '%target-triple' | sed -E -e 's/macosx10.(9|10).*/macosx10.11/') -sdk %sdk -I %S/Inputs/ObjCClasses/ %t/ObjCClasses.o %s -o %t/a.out
+// RUN: %target-swiftc_driver -target $(echo '%target-triple' | sed -E -e 's/macosx10.(9|10).*/macosx10.11/') -sdk %sdk -I %S/Inputs/ObjCClasses/ %t/ObjCClasses.o %s -o %t/a.out
```

Comment:
Can you remove the `-target` parameter now that you are using `%target-swiftc_driver`?

Reply:
Good point -- I left that in place because it documents that [some ObjC metadata are not emitted for older targets](https://github.com/apple/swift/pull/2350/files#diff-8b7b599c8f7b61f9524247a0b32c39cc9cc9f3290df881b94bea8f20688c8dc0R1646), but noticing now this does not add any value, given `macosx10.13` is the [default deployment target configured in build-script](https://github.com/apple/swift/blob/9019d07fd71e7e405f561936483b51e5369e385c/utils/build_swift/build_swift/defaults.py#L54), and the lowest deployment target allowed by Xcode 14.2 (the version we currently advertise on [https://](https://ci.swift.org))

Reply:
This is addressed with ea5d41632a25a9756d920effa7b23c99d7617da3